app/javascript/dashboard/components/ui/TimeAgo.vue

```diff
@@ -19,20 +28,40 @@ export default {
       type: Boolean,
       default: true,
     },
-    timestamp: {
+    lastActivityTimestamp: {
+      type: [String, Date, Number],
+      default: '',
+    },
+    createdAtTimestamp: {
       type: [String, Date, Number],
       default: '',
     },
   },
   data() {
     return {
-      timeAgo: this.dynamicTime(this.timestamp),
+      lastActivityAtTimeAgo: this.dynamicTime(this.lastActivityTimestamp),
+      createdAtTimeAgo: this.dynamicTime(this.createdAtTimestamp),
       timer: null,
     };
   },
+  computed: {
+    lastActivityTime() {
+      return this.shortTimestamp(this.lastActivityAtTimeAgo);
+    },
+    createdAtTime() {
+      return this.shortTimestamp(this.createdAtTimeAgo);
+    },
+    tooltipText() {
+      return `Created at: ${this.createdAtTimeAgo}
```

Comment:
If we are formatting this as "X days ago" we should not use "Created at:"

The UI can be more conversational like

Created 5 days ago, last activity 3 days ago

And we can put a limit of 3 months, if the date is older than 3 months, we show the exact date on hover, something like

Created on 5th August 2022, last activity today

Because after a point it's difficult to mentally infer dates and months

CC: @nithindavid what do you think?

Reply:
Totally makes sense. Can we fall back to normal (MM/DD/YY)  if it's created before a month?  I find the `2 months ago` not usefull after a point as, 1 month 28 days would also be shown as more than a month ago; which doesn't contribute to the visual sorting logic + slows the urgency one might need to do.

Reply:
@nithindavid @scmmishra Thoughts on this? If the conversation is created before a month and the user activity is not there for more than one month, it will show in `MM/DD/YYYY.` Else, it will show in the ago format.


https://user-images.githubusercontent.com/64252451/213091983-44a28242-91b9-4bef-9e24-c4ff782b5bf4.mov

Reply:
@scmmishra @nithindavid Updated the PR with the last commented changes.

Reply:
Looks great!